app/v2/controllers/validators/ListHistoricUkPropertyPeriodSummariesValidatorFactory.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v2.controllers.validators
+
+import api.controllers.validators.Validator
+import api.controllers.validators.resolvers.ResolveNino
+import api.models.errors.MtdError
+import cats.data.Validated
+import config.AppConfig
+import v2.models.request.listHistoricUkPropertyPeriodSummaries.ListHistoricUkPropertyPeriodSummariesRequestData
+
+import javax.inject.{Inject, Singleton}
+
+@Singleton
+class ListHistoricUkPropertyPeriodSummariesValidatorFactory @Inject() (appConfig: AppConfig) {
```

Comment:
the AppConfig isn't being used so we can update this
```suggestion
class ListHistoricUkPropertyPeriodSummariesValidatorFactory {
```